service/src/integrationTest/kotlin/fi/espoo/evaka/shared/db/SchemaConventionsTest.kt

```diff
@@ -0,0 +1,720 @@
+                .toSet()
+        assertEquals(permittedViolations, violations)
+    }
+
+    @Test
+    fun `update timestamp should be 'timestamp with time zone' and NOT NULL`() {
+        val permittedViolations =
+            setOf(
+                Column(
+                    ColumnRef("backup_care", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("backup_messaging_blocklist", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("daycare_group_placement", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("decision", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("fridge_child", "created_at"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("guardian", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("invoice", "created_at"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("language_emphasis", "created"),
+                    "timestamp without time zone",
+                    nullable = false,
+                ),
+                Column(
+                    ColumnRef("mobile_device_push_group", "created_at"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("placement_plan", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("staff_attendance", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("voucher_value_decision", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+                Column(
+                    ColumnRef("voucher_value_report_snapshot", "created"),
+                    "timestamp with time zone",
+                    nullable = true,
+                ),
+            )
+        val violations =
+            columns
+                .filter { it.ref.columnName == "created" || it.ref.columnName == "created_at" }
```

Comment:
copy paste error? should be
`.filter { it.ref.columnName == "updated" || it.ref.columnName == "updated_at" }`